Dockerfile

```diff
@@ -45,6 +45,15 @@ RUN --mount=target=. \
     xx-verify /usr/bin/hugo
 EOT
 
+# dart-sass downloads the dart-sass runtime dependency
+FROM alpine:${ALPINE_VERSION} AS dart-sass
+ARG TARGETARCH
+ARG DART_SASS_VERSION=1.79.3
```

Comment:
Can you make these Dart `ARG` vars into ENV. I understand the ... flexibility, but it just adds work to the documentation/testing pile.

Other than that this looks/works great.